build_tools/bazel/iree.bazelrc

```diff
@@ -15,6 +15,9 @@ build --experimental_guard_against_concurrent_changes
 build --compilation_mode=opt
 # Used in TensorFlow, so we have to enable it here as well.
 common --experimental_repo_remote_exec
+# TensorFlow uses cc_shared_library, which was added behind this experimental
+# flag in Bazel 5.1.0. Since we depend TensorFlow, we have to enable it too.
+build --experimental_cc_shared_library
```

Comment:
We should be able to put this only in iree-tf.bazelrc

Reply:
Want to try that in this PR, or try moving `common --experimental_repo_remote_exec` and maybe some other flags there in a follow-up?

Reply:
I moved the new flag but left the old flags where they were. Hopefully this still passes our CI builds 🤞